01-ganga-with-cmake.md

```diff
@@ -82,57 +76,41 @@ $ make install
 ```
 
 This will create a folder called `InstallArea`. Ganga needs to 
-know about this folder a couple of others to use the local project. We will create [symbolic links][symlink] 
-to these folders in the place that Ganga looks.
-
-```shell
-$ mkdir -p ~/cmtuser/DaVinci_v40r1p3
-$ cd ~/cmtuser/DaVinci_v40r1p3
-$ ln -s ~/DaVinciDev_v40r1p3/cmt
-$ ln -s ~/DaVinciDev_v40r1p3/DaVinciDevSys
-$ ln -s ~/DaVinciDev_v40r1p3/InstallArea
-$ ls -la
-drwxr-xr-x.  2 username z5 2048 May 19 09:05 .
-drwxr-xr-x.  2 username z5 2048 May 19 09:05 ..
-lrwxr-xr-x.  1 username z5   58 May 19 09:05 cmt -> /afs/cern.ch/user/a/apearce/DaVinciDev_v40r1p3/cmt
-lrwxr-xr-x.  1 username z5   58 May 19 09:05 DaVinciDevSys -> /afs/cern.ch/user/a/apearce/DaVinciDev_v40r1p3/DaVinciDevSys
-lrwxr-xr-x.  1 username z5   58 May 19 09:05 InstallArea -> /afs/cern.ch/user/a/apearce/DaVinciDev_v40r1p3/InstallArea
-```
+know about the local project.
```

Comment:
These two sentences are quite incongruous now:

> This will create a folder called `InstallArea`. Ganga needs to know about the local project.

Maybe the last sentence can be "This is the folder that Ganga will ship to the Grid." or something (if that's true).